EIPS/eip-7702.md

```diff
@@ -44,19 +44,22 @@ rlp([chain_id, nonce, max_priority_fee_per_gas, max_fee_per_gas, gas_limit, dest
 authorization_list = [[chain_id, address, nonce, y_parity, r, s], ...]
 ```
 
-Transaction is considered invalid if authorization list items can't be decoded as:
-
-* `chain_id`: unsigned 256-bit integer.
-* `nonce`: unsigned 64-bit integer.
-* `address`: 20 bytes array.
-* `y_parity`: Value 0 or 1.
-* `r`: unsigned 256-bit integer.
-* `s`: unsigned 256-bit integer and value less or equal than `secp256k1n/2`, specified in [EIP-2](./eip-2.md).
-
 The fields `chain_id`, `nonce`, `max_priority_fee_per_gas`, `max_fee_per_gas`, `gas_limit`, `destination`, `value`, `data`, and `access_list` of the outer transaction follow the same semantics as [EIP-4844](./eip-4844.md). *Note, this means a null destination is not valid.*
 
 The `authorization_list` is a list of tuples that store the address to code which the signer desires to execute in the context of their EOA. The transaction is considered invalid if the length of `authorization_list` is zero.
 
+The transaction is also considered invalid when any field in an authorization
+tuple cannot fit within the following bounds:
+
+```python
+assert auth.chain_id < 2**256
+assert auth.nonce < 2**64
```

Comment:
Have a slight nit about this. (Sorry, just noticed after-merge). By https://eips.ethereum.org/EIPS/eip-2681 should this not be `2**64 - 1`? Or is this checkout outside tx validity, but instead inside EVM?

Reply:
It is, but specific to the tx sender. The idea here is to only give numeric boundaries here and leave interpretation to other subsystems.

Reply:
> It is, but specific to the tx sender.

The nonce of authority is also incremented during delegation setting, so it is important to check `nonce < 2**64 - 1` here.

I've made this change in https://github.com/ethereum/EIPs/pull/8905